tests/shared/src/test/scala/cats/tests/TraverseFilterSuite.scala

```diff
@@ -38,6 +38,19 @@ abstract class TraverseFilterSuite[F[_]: TraverseFilter](name: String)(implicit
 
   implicit def T: Traverse[F] = implicitly[TraverseFilter[F]].traverse
 
+  test(s"TraverseFilter[$name].mapAccumulateFilter") {
```

Comment:
Looks like it does not check the default implementation (based on `State`), does it?
Except maybe one for `Stream`, but I wouldn't count on it.

For testing default implementations we usually use `ListWrapper` from **testkit**:
[ListWrapper.scala](https://github.com/typelevel/cats/blob/main/testkit/src/main/scala/cats/tests/ListWrapper.scala)

Reply:
My apologies for the delay – I was snowed under a bit. Actually, there's `TraverseFilter` for `ListWrapper`:
https://github.com/typelevel/cats/blob/47fbad764ad8026d4e9f8999c7f6233c666f0df8/testkit/src/main/scala/cats/tests/ListWrapper.scala#L80-L90

To test the default implementation you can either to call it directly:
```scala
ListWrapper.traverseFilter.mapAccumulateFilter(...)
```
or make it an implicit in the scope:
```scala
implicit val listWrapperTraverseFilter: TraverseFilter[ListWrapper] = ListWrapper.traverseFilter
```
And then you can work with `TraverseFilter` for `ListWrapper` as usual.